hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTable.scala

```diff
@@ -38,7 +38,7 @@ class TestAlterTable extends TestHoodieSqlBase {
              |  ts long
              |) using hudi
              | location '$tablePath'
-             | options (
+             | tblproperties (
```

Comment:
here means users would not use options?

Reply:
`options` will be saved to `storage.properties` in hive metastore. and `tblproperties` to `properties`.
Using `tblproperties` is a more proper way. Also for compatibility, we  still support `options`.